components/AppHeader.tsx

```diff
@@ -7,20 +7,46 @@ import LinkIcon from "./icons/LinkIcon";
 import EditIcon from "./icons/EditIcon";
 import { useAccount } from "wagmi";
 import UpadateApplication from "./ApplicationForm";
+import { updateApplication } from "@/lib/actions";
+import Loading from "./ApplicationForm/Loading";
 
 type AppHeaderProps = {
   application: IApplication;
 };
 
+const ALLOW_LIST = ["0xd836D2c9a6e014c2056093BdC4FaA7343CAe80c9"];
```

Comment:
I would say, move this to a safer/secret location.

Reply:
Should we add this to a database, or will an ENV be enough?